ecosystem/sep-0010.md

```diff
@@ -46,7 +46,9 @@ The authentication flow is as follows:
 1. The **Client** verifies that if the transaction has a Manage Data operation with key `web_auth_domain` that it has:
    1. Source account set to the **Server Account**.
    1. Value set to the **Server**'s domain that the client requested the challenge from.
-1. The **Client** verifies that if the transaction has other operations they are Manage Data operations that all have their source accounts set to the **Server Account**
+1. The **Client** verifies that if the transaction has other operations they are Manage Data operations and that their source account is set to:
+   1. The **Server Account** if the Manage Data operation key is NOT set to `client_domain`
+   1. The **Client Domain Account** if the Manage Data operation key is set to `client_domain`
```

Comment:
I think this list can be simplified a little such that if we add other manage data operations with other rules they can be inserted here.
```suggestion
   1. The **Client Domain Account** if the Manage Data operation key is set to `client_domain`
   1. Otherwise, the **Server Account**.
```